### ISSUE TITLE
What is x in the first (and only) example?

### ISSUE THREAD

**Issue by OnkelTem** [python-trio/asyncclick#13]
This is example's code:

```python

import anyio
import asyncclick as click

@click.command()
@click.option("--count", default=1, help="Number of greetings.")
@click.option("--name", prompt="Your name", help="The person to greet.")
async def hello(count, name):
    """Simple program that greets NAME for a total of COUNT times."""
    for _ in range(count):
        if x: await anyio.sleep(0.1)
        click.echo(f"Hello, {name}!")

if __name__ == '__main__':
    hello(_anyio_backend="trio")  # or asyncio, or curio

```

So what is `x` I wonder?

**Comment by smurfix** [comment]
That should be `for x in range…`. Will fix, thanks.

**Comment by OnkelTem** [comment]
Good!

**Comment by smurfix** [comment]
Fixed in current main branch (tracking 8.0)
